heudiconv/bids.py

```diff
@@ -196,6 +196,8 @@ def populate_bids_templates(
         op.join(path, "README"),
         "TODO: Provide description for the dataset -- basic details about the "
         "study, possibly pointing to pre-registration (if public or embargoed)",
+        # TODO: get from schema
+        glob_suffixes=[".md", ".txt", ".rst"],
```

Comment:
Shouldn't you also add the empty string as a possible suffix?

Reply:
d'oh, right -- there could be indeed a suffix less match, good catch!